plugins/org.preesm.model.pisdf/src/org/preesm/model/pisdf/util/topology/PiSDFSuccessorSwitch.java

```diff
@@ -112,6 +112,15 @@ public Boolean casePiGraph(final PiGraph graph) {
     private static final long serialVersionUID = 4039813342986334156L;
   }
 
+  /**
```

Comment:
Move these classes in their own files, and rename them to better reflect what they do.

Also what defines a "simple path" ?

Reply:
I renamed it IsThereALongPathSwitch (it should be clearer) :
A Long Path is a path that is composed of more than one fifo. Two actors are not mergeable if a path is composed of more than one fifo.